db/migrate/20160106233632_convert_classification_name_to_column.rb

```diff
@@ -0,0 +1,20 @@
+class ConvertClassificationNameToColumn < ActiveRecord::Migration
+  class Classification < ActiveRecord::Base
+    belongs_to :tag
+  end
+
+  class Tag < ActiveRecord::Base
+  end
+
+  def up
+    add_column :classifications, :name, :string
+    Classification.all.each do |classification|
+      next unless classification.tag
```

Comment:
I don't know how big the classifications table is, so my comment might be irrelevant. Can we preload `tag` association there? So that each call to `classification.tag` would not result in an additional request to the database.

```ruby
Classification.includes(:tag).each do |classification|
  next unless classification.tag
  ...
```

Or even filter out tagless classifications on the database side:

```ruby
Classification.includes(:tag).where.not(classifications: { tag_id: nil }).each do |classification|
  ...
```